### ISSUE TITLE
Getting "Press enter to continue" prompt. Prototype is not wrapping correctly in the command line.

### ISSUE THREAD

**Issue by artemvmin** [majutsushi/tagbar#505]

**Comment by artemvmin** [comment]
Pausing over a method causes the proto to be written to the command line, which wraps and causes the following prompt:
"Press ENTER or type command to continue"

As a short term solution, I would recommend providing a way to disable this "hover->show proto" functionality.

**Comment by artemvmin** [comment]
I'm running Vim 8.0.

**Comment by artemvmin** [comment]
I set g:tagbar_silent as a temporary fix.

**Comment by alerque** [comment]
Yes, that's somewhat less than ideal and a PR that added an option flag to disable this "feature" would be lovely if anybody wants to contribute such a thing!
